src/main/java/xades4j/production/BasicSignatureOptions.java

```diff
@@ -24,12 +24,33 @@
  */
 public final class BasicSignatureOptions
 {
+    private boolean checkKeyUsage = true;
     private SigningCertificateMode includeSigningCertificateMode;
     private boolean includeSubjectName;
     private boolean includeIssuerSerial;
     private boolean includePublicKey;
     private boolean signKeyInfo;
 
+    /**
+     * Configures whether to check that the keyUsage of the signing certificate
+     * allows use for signing before creating a signature. If enabled (the default)
+     * signing will fail if the keyUsage of the certificate does not allow signing.
+     * You should only disable this for testing.
+     *
+     * @param enabled {@code true} to enable the check, {@code false to disable}
```

Comment:
Detail: "to disable" should be outside the code tag

Reply:
Fixed, thank you.